xray/audit/commonutils.go

```diff
@@ -123,10 +123,13 @@ func GetModule(modules []*services.GraphNode, moduleId string) *services.GraphNo
 	return nil
 }
 
-func LogExecutableVersion(executable string) {
-	// Get executable version and print to log if possible
-	verString, _ := exec.Command(executable, "--version").CombinedOutput()
-	if len(verString) > 0 {
-		log.Debug(fmt.Sprintf("Used %q version: %s", executable, strings.TrimSpace(string(verString))))
+// Get executable version and print to log if possible.
+func GetAndLogExecutableVersion(executable string) (version string, err error) {
```

Comment:
Since this function only returns the executable version, and the logging is in debug log level, I suggest renaming this function to `GetExecutableVersion`.
Let's change the fnction description accordingly, and also note that the function uses the `--version` flag, which isn't supported by by any executable type.